app/rtc_s1.py

```diff
@@ -1222,6 +1227,18 @@ def get_rtc_s1_parser():
                         default=False,
                         help='Enable full formatting of log messages')
 
+    parser.add_argument('--keep-scratch',
+                        dest='keep_scratch',
+                        action='store_true',
+                        default=False,
+                        help='keep the files in scratch directory')
+
+    parser.add_argument('--timestamp',
+                        dest='timestamp',
+                        type=str,
+                        default='',
+                        help='timestamp for temporary directory')
```

Comment:
We can remove this

Reply:
uhm... This argument is necessary to force all the burst processes to use the same temporary directory. Otherwise, the procedure following the burst processing gets too much complicated.

Reply:
I think that's fine that all processes share the same time stamp and temporary directory, but I don't think we should expose the timestamp to the user.

Reply:
@gshiroma Solving this issue by importing `rtc_s1` and executing `run(cfg) inside `process_runconfig()`, rather than calling `rtc_s1.py` from terminal.

Reply:
irrelevant with the latest version of the code, so I will mark this conversation as resolved.